site/en/docs/extensions/mv3/promises/index.md

```diff
@@ -199,24 +203,28 @@ async function openTabOnRight() {
   // When not wrapped in try/catch, errors thrown in an async
   // function will propagate down the promise chain
   let tabs = await chrome.tabs.query(queryOptions);
-
   if (!tabs.length) return;
+
   let tab = await chrome.tabs.create({
     url: 'https://example.com',
     index: tab[0].index + 1,
   });
-
   if (!tab) return;
+
   console.log('tab created', tab);
   return tab;
 }
 ```
 
 {% Aside %}
-Note that `await` is only valid in async functions and the top-level bodies of modules.
+
+Note that `await` is only valid in async functions and the top-level bodies of modules. You can load
```

Comment:
```suggestion
Note that `await()` is only valid in async functions and the top-level bodies of modules. You can load
```

Reply:
@jpmedley, I'm not sure this suggestion applies. In this case [`await`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await) is an operator that modifies an expression to await the resolution of a Promise. If this was intentional, would you mind explaining how it applies here?

Reply:
Brain not working that day. Sorry.